src/graphql-schema.js

```diff
@@ -95,32 +95,24 @@ export const resolvers = {
       // if (user && user.id) {
       //   throw new Error('Already logged in.')
       // }
-
       const session = driver.session()
-      const res = await session.run('MATCH (u:User {email: "' + email + '"}) RETURN u.id, u.slug, u.name, u.avatar, u.email, u.password, u.role LIMIT 1')
-      let u = res.records[0]._fields ? zipObject([
-        'id',
-        'slug',
-        'name',
-        'avatar',
-        'email',
-        'password',
-        'role'
-      ], res.records[0]._fields) : null
-      if (u) {
-        if (await bcrypt.compareSync(password, u.password)) {
-          delete u.password
-          u.avatar = fixUrl(u.avatar)
-          return Object.assign(u, {
-            token: generateJwt(u)
+      return session.run(
+        'MATCH (user:User {email: "' + email + '"}) ' +
```

Comment:
@Gerald1614 do you know if we have a cypher injection vulnerability here?

Reply:
I opened a seperate Issue for the Injection/Sanitization issue. #95

Reply:
I did not pay attention because this piece was already there.  but we shoudl use varibales to inject data in teh query. the request should look like:
```'MATCH (user:User {email: $userEmail}) ' +
        'RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1', { userEmail: email })```

Reply:
this is the recommended way when dealing with cypher and variables. we should use this.  Graphql playground includes a place to inject those data as well.